packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -50,6 +50,8 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)captureToFile:(FLTThreadSafeFlutterResult *)result API_AVAILABLE(ios(10));
 - (void)close;
 - (void)startVideoRecordingWithResult:(FLTThreadSafeFlutterResult *)result;
+- (void)startVideoRecordingWithResult:(FLTThreadSafeFlutterResult *)result
+                messengerForStreaming:(nullable NSObject<FlutterBinaryMessenger> *)messenger;
```

Comment:
can you add a comment saying that the optional param is for concurrent streaming during video recording

Reply:
Sure, done.